requirements.txt

```diff
@@ -0,0 +1,2 @@
+psycopg2-binary==2.7.5
+Django==2.1.5
```

Comment:
Tiny request: can we use 2.1.7?

Reply:
Postgres is fine, but im keen on ensuing we do not loose Django's database agnosticism.